jenkins_api_client.gemspec

```diff
@@ -23,8 +23,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'nokogiri', '~> 1.6'
   s.add_dependency 'thor', '>= 0.16.0'
   s.add_dependency 'terminal-table', '>= 1.4.0'
-  s.add_dependency 'mixlib-shellout', '>= 1.1.0'
+  s.add_dependency 'mixlib-shellout', '~> 2.0.0'
```

Comment:
This is highly restrictive and wouldn't allow any minor bumps. Not sure this is what is intended?